doc/node_help/scd4x.rst

```diff
@@ -3,7 +3,7 @@ Sensirion SCD4X CO2 sensor
 
 ..  code-block:: cpp
 
-    scd4x(name)[.i2c(sda,scl)][.with_address(i2c_address)][.with_filter(filter)];
+    scd4x(name[, temp_comp][, altitude])[.i2c(sda,scl)][.with_address(i2c_address)][.with_filter(filter)];
```

Comment:
it should be [, temp_comp[, altitude]] as you cannot specify only the altitude (without specifying the temperature - cpp restriction - in python that works).

Reply:
done